src/longRange/Homogeneous.cpp

```diff
@@ -133,7 +133,7 @@ void Homogeneous::calculateLongRange() {
 	double UpotCorr = UpotCorrLJ + MySelbstTerm;
 	double VirialCorr = VirialCorrLJ + 3. * MySelbstTerm;
 
-	global_log->info() << "Far field terms: U_pot_correction  = " << UpotCorr << " virial_correction = " << VirialCorr
+	global_log->debug() << "Far field terms: U_pot_correction  = " << UpotCorr << " virial_correction = " << VirialCorr
```

Comment:
why move this to debug?

Reply:
because it is now written in every time step and otherwise clutters everything.